src/api.ts

```diff
@@ -12,5 +12,11 @@ export async function getPeople(): Promise<Person[]> {
   // keep this delay for testing purpose
   return wait(500)
     .then(() => fetch(API_URL))
-    .then(response => response.json());
+    .then(response => {
+      if (!response.ok) {
+        throw new Error('Something went wrong!');
```

Comment:
The error message 'Something went wrong!' is quite generic. Consider providing more specific information about the error, such as the status code or a more descriptive message, to aid in debugging.